hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieCompactionConfig.java

```diff
@@ -106,6 +106,12 @@ public class HoodieCompactionConfig extends HoodieConfig {
       .withDocumentation("Only if the log file size is greater than the threshold in bytes,"
           + " the file group will be compacted.");
 
+  public static final ConfigProperty<Long> COMPACTION_LOG_FILE_LENGTH_THRESHOLD = ConfigProperty
+      .key("hoodie.compaction.logfile.length.threshold")
+      .defaultValue(0L)
```

Comment:
Should this be set to a reasonable value like `5` for example?  Otherwise, it falls back to the behavior where all file groups are compacted.

Reply:
Got it.  That makes sense.  Then do we even need this config?  The new compaction strategy prioritizes the compaction of file groups with more log files, and should include all file groups for compaction nevertheless.

Reply:
Got it.  Let’s keep it then.